pymatgen/entries/computed_entries.py

```diff
@@ -11,26 +11,189 @@
+    schemes such as the Aqueous energy referencing scheme.
+    """
+    def __init__(self, value, name="Constant energy adjustment", cls=None, description="Constant energy adjustment"):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+            name: str, human-readable name of the energy adjustment.
+                (Default: Constant energy adjustment)
+            cls: dict, Serialized Compatibility class used to generate the energy adjustment. (Default: None)
+            description: str, human-readable explanation of the energy adjustment.
+        """
+        description = description + " ({:.3f} eV)".format(value)
+        super().__init__(value, name, cls, description)
+        self._value = value
+
+    @property
+    def value(self):
+        """
+        Return the value of the energy correction in eV.
+        """
+        return self._value
+
+    @value.setter
+    def value(self, x):
+        self._value = x
+
+    def _normalize(self, factor):
+        self._value /= factor
+
+
+class ManualEnergyAdjustment(ConstantEnergyAdjustment):
+    """
+    A manual energy adjustment applied to a ComputedEntry.
+    """
+    def __init__(self, value):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+        """
+        name = "Manual energy adjustment"
+        description = "Manual energy adjustment"
+        super().__init__(value, name, cls=None, description=description)
+
+
+class CompositionEnergyAdjustment(EnergyAdjustment):
+    """
+    An energy adjustment applied to a ComputedEntry based on the atomic composition.
+    Used in various DFT energy correction schemes.
+    """
+    def __init__(self, adj_per_atom, n_atoms, name, cls=None, description="Composition-based energy adjustment"):
+        """
+        Args:
+            adj_per_atom: float, energy adjustment to apply per atom, in eV/atom
+            n_atoms: float or int, number of atoms
+            name: str, human-readable name of the energy adjustment.
+                (Default: "")
+            cls: dict, Serialized Compatibility class used to generate the energy adjustment. (Default: None)
+            description: str, human-readable explanation of the energy adjustment.
+        """
+        self._value = adj_per_atom
+        self.n_atoms = n_atoms
+        self.cls = cls if cls else {}
+        self.name = name
+        self.description = description + " ({:.3f} eV/atom x {} atoms)".format(self._value,
+                                                                               self.n_atoms
+                                                                               )
+
+    @property
+    def value(self):
+        """
+        Return the value of the energy adjustment in eV.
+        """
+        return self._value * self.n_atoms
+
+    def _normalize(self, factor):
+        self.n_atoms /= factor
+
+
+class TempEnergyAdjustment(EnergyAdjustment):
```

Comment:
Sorry to nitpick, but can you change this to `TemperatureEnergyAdjustment`?

Reply:
👍